### ISSUE TITLE
Necromancer Damage

### ISSUE THREAD

**Issue by FallFlatMatt** [BleedingOptimism/AlexTTTIssues#39]
Necromancer is too powerful and needs a damage reduction.

**Comment by Mokerak** [comment]
Agreed, they should rely on their zombies for protection, but they shouldnt deal no damage at all, otherwise they cant protect themselves from traitors.
